src/main/java/dev/snowdrop/release/App.java

```diff
@@ -134,6 +139,42 @@ public void link(
         service.linkIssue(fromIssue, toIssue);
     }
 
+    @CommandLine.Command(
+        name = "new-maj-min",
```

Comment:
I don't like the name......what about `init-release`, `prepare-release-files`....

Reply:
I'm OK with changing the name, but I think neither `init-release` nor `prepare-release-files` are good candidates because they're ambiguous as for the tasks being executed. These tasks are for "_preparing/initializing_" a release but are only required when we're dealing with a new Major.Minor version.

Reply:
How about `init-for-new-maj-min-release` which reflects that it's a repository initialization and it's for a new Major.Minor version?

Reply:
Personally, I think this command should be removed soon so I don't really care about how it's named… :)

Reply:
I like the name of the command as proposed by Chris

Reply:
`new-build-config-version` is Ok for me

Reply:
We have 3 yesses!!!!